tests/test_cases/test_cocotb/test_tests.py

```diff
@@ -193,37 +192,3 @@ async def test_func():
 
     cocotb.start_soon(test_func())
     await NullTrigger()
-
-
-@cocotb.test(expect_fail=True)
-async def test_pytest_raises_fail_expect_fail(dut):
-    with pytest.raises(AssertionError):
-        assert True
-
-
-@cocotb.test(expect_fail=True)
-async def test_base_exception_expect_fail(dut):
-    raise MyBaseException
-
-
-@cocotb.test(expect_fail=True)
-async def test_base_exception_in_task_expect_fail(dut):
-    async def test_func():
-        raise MyBaseException
-
-    cocotb.start_soon(test_func())
-    await NullTrigger()
```

Comment:
I think these should stay in with `expect_error`, to confirm independently from `pytest` that we catch exceptions derived from `BaseException`.